barretenberg/cpp/src/barretenberg/stdlib/primitives/biggroup/biggroup_goblin.hpp

```diff
@@ -39,7 +39,8 @@ element<C, Fq, Fr, G> element<C, Fq, Fr, G>::goblin_batch_mul(const std::vector<
     // Loop over all points and scalars
     size_t num_points = points.size();
     for (size_t i = 0; i < num_points; ++i) {
-        auto& point = points[i];
+        auto tmp_point = points[i].normalize();
```

Comment:
this was needed to get tests to pass. Not exactly sure why

Reply:
Seems possible it has something to do with [this](https://github.com/AztecProtocol/aztec-packages/blob/d641a4bcce91c785e84cdfccb57a8ae120477e39/barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/op_queue/ecc_op_queue.hpp#L70)

Reply:
hm, I don't see why